content/docs/sidebar.json

```diff
@@ -87,7 +87,11 @@
         "slug": "sharing-data-and-model-files"
       },
       "data-registries",
-      "shared-development-server"
+      "shared-development-server",
+      {
+        "label": "CI/CD for Machine Learning",
+        "slug": "ci-cd-ml"
```

Comment:
💅 `ci-for-ml`? (renders in the URL, may have soooome SEO impact).

Reply:
Would `ci-cd-for-ml` be better? Or even `ci-cd-for-machine-learning`?

Reply:
Yes. I think `ml` is fine but 👍 either one. Thanks!

Reply:
done